### PR TITLE
setup basic component structure and formatting

### PR DESCRIPTION
**Pull request by claireolmstead**
# Problem

As a developer, I want to have the component structure roughly setup, along with basic formatting, so that when we build out the first features, there is a sense of organization.
[link to Pivotal Tracker #12345678](https://www.pivotaltracker.com/story/show/12345678)

# Solution
- Build the general component structure
- Add formatting only
 - No brand-influenced styling
- Add tests to make sure components render

## Change summary:

- Created the following empty components
 - Header
 - Feed
 - NewPost
 - PostList
 - Profile
- Created tests to make sure they render
- Added styling to for layout and to show layout

## Steps to Verify:

1. Run EC and see that everything is rendering correctly
2. Check that the component structure is correct

## Screenshots (optional):
Screen Shot 2021-03-19 at 4 17 10 PM

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -1,43 +1,10 @@
...
-  height: 40vmin;
-  pointer-events: none;
-}
-
-.App__header {
-  @include headline-text;
-
-  background-color: $gray-28;
-  min-height: 100vh;
+.Content__block {
```
Comment (claireolmstead) [inline src/App.scss:7]: Is it weird to put this styling here? It matches the file format, but feels a little weird under the imports.  What's the right call?

File: .gitignore [hunk 2]
```diff
@@ -24,3 +24,5 @@ yarn-error.log*
 
 .idea
 junit.xml
+src/App.css
+src/App.css.map
```
Comment (wilwade) [inline .gitignore:28]: If we git rm these files, we do not need to ignore them

File: src/App.scss (hunk 1 above)
Comment (wilwade) [reply src/App.scss:7]: I'd likely name it `.App__content` instead, that way it follows more of BEM, but having this file as the App component's styles is fine by me.

**Review by wilwade — CHANGES_REQUESTED** [review]
Two small notes.

**Review by wilwade — APPROVED** [review]

**Pull request merged**